ci-operator/config/opendatahub-io/notebooks/opendatahub-io-notebooks-main.yaml

```diff
@@ -940,6 +952,65 @@ tests:
       IMAGE_REPO: workbench-images
       RELEASE_VERSION: amd-jupyter-minimal-c9s-python-3.9
     workflow: opendatahub-io-ci-image-mirror
+- as: notebook-amd-jupyter-ds-ubi9-python-3-9-image-mirror
+  postsubmit: true
+  steps:
+    dependencies:
+      SOURCE_IMAGE_REF: amd-jupyter-datascience-ubi9-python-3.9
+    env:
+      IMAGE_REPO: workbench-images
+      IMAGE_TAG: YearIndex
+      RELEASE_VERSION: amd-jupyter-datascience-ubi9-python-3.9
+    workflow: opendatahub-io-ci-image-mirror
```

Comment:
For now, we can drop the AMD data-science and minimal, as we are not gonna provide them in.
leaving this for future.

Remove these lines, and the minimal as well, as we don't really provide them to the user.